src/storage.rs

```diff
@@ -1,8 +1,42 @@
-pub struct FileStorage {}
+use miniserde::{json, Deserialize, Serialize};
+use std::fs;
+use std::path::Path;
+
+const FILENAME: &str = "filestorage.json";
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FileStorage {
+    username: String,
+    password: String,
```

Comment:
Do we want to store these at all? Storing the password seems like a security concern at least.